App/CrisisCleanup/Sources/Feature/Cases/CasesComponent.swift

```diff
@@ -8,7 +8,11 @@ protocol CasesViewBuilder {
 class CasesComponent: Component<AppDependency>, CasesViewBuilder {
     lazy var casesViewModel: CasesViewModel = {
         CasesViewModel(
+            appEnv: dependency.appEnv,
+            accountDataRepository: dependency.accountDataRepository,
             incidentSelector: dependency.incidentSelector,
+            appVersionProvider: dependency.appVersionProvider,
+            authEventBus: dependency.authEventBus,
```

Comment:
These four properties are not necessary for `CasesViewModel` and can be removed.